cms/djangoapps/contentstore/views/item.py

```diff
@@ -805,8 +812,14 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "format": xblock.format,
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
-        "override_type": override_type,
+        "actions": xblock_actions,
+        "explanatory_message": explanatory_message
     }
+
+    # Entrance exam subsection should be hidden.
+    if xblock.category == 'sequential' and getattr(parent_xblock, "is_entrance_exam", False):
```

Comment:
But it is possible that there is a sequential that is not related to an entrance exam. So is it possible that there is a sequential not related to an entrance exam, and parent_xblock is None? What is the value of getattr(parent_xblock, "is_entrance_exam", False) if parent_xblock is None?